.github/workflows/miniwdl-test.yml

```diff
@@ -0,0 +1,145 @@
+name: MiniWDL Spec Conformance - WDL 1.2
+
+on:
+  push:
+
+jobs:
+  conformance:
+    runs-on: ubuntu-latest
+    name: Run WDL 1.2 Spec Conformance Tests
+
+    steps:
+      - name: Checkout WDL spec repo (wdl-1.2)
+        uses: actions/checkout@v4
+        with:
+          repository: openwdl/wdl
+          ref: wdl-1.2
+          path: wdl-spec
+
+      - name: Checkout current repo
+        uses: actions/checkout@v4
+        with:
+            fetch-depth: 0        # fetch all commits and refs
+            # For pull_request events, make sure the PR SHA is checked out
+            ref: ${{ github.event.pull_request.head.sha || github.sha }}
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.10'
+
+      - name: Install MiniWDL (main branch) and dependencies
+        run: |
+          pip install git+https://github.com/chanzuckerberg/miniwdl.git@main
+          pip install junit-xml subby
+
+      - name: Clone WDL tests
+        run: |
+          git clone https://github.com/openwdl/wdl-tests.git
+
+      - name: Extract tests
+        run: |
+          python wdl-tests/scripts/extract_tests.py -i ${{ github.workspace }}/SPEC.md -d ${{ github.workspace }}/tests/data/ -O miniwdl_test -v 1.2
+
+      - name: Run tests and capture results
+        run: |
+          mkdir -p miniwdl_tests/artifacts
+          python wdl-tests/scripts/run_tests_miniwdl.py -T ${{ github.workspace }}/miniwdl_test -c ${{ github.workspace }}/miniwdl_test/test_config.json -D ${{ github.workspace }}/miniwdl_test/data -O miniwdl_results | tee miniwdl_tests/artifacts/result.log
```

Comment:
Maybe an echo to say that the tests are running before this (and for all runners)?

Reply:
Good idea.